pages/platform/databases/databases_06_restore_backup/guide.de-de.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Restore a backup
+slug: restore-backup
+excerpt: Find out how to restore a backup
+section: General guides
+order: 060
+routes:
+    canonical: 'https://docs.ovh.com/gb/en/publiccloud/databases/restore-backup'
+---
+
+**Last updated May 4th, 2022**
+
+## Objective
+
+OVHcloud Databases as-a-service (DBaaS) allow you to focus on building and deploying cloud applications while OVHcloud takes care of the database infrastructure and maintenance. 
+
+**This guide explains how to restore a backup of a database solution in the OVHcloud Control Panel.**
+
+We continuously improve our offers. You can follow and submit ideas to add to our roadmap at <https://github.com/ovh/public-cloud-roadmap/projects/2>.
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.de/&ovhSubsidiary=de)
+- A [Public Cloud database service](https://www.ovhcloud.com/de/public-cloud/databases/) up and running
+
+## Before we begin
+In this guide we will use a Postgresql database engine as an example, but the procedure is exactly the same for all other engines.
+
+You can learn more about how backups works on the [Automated backups guide](https://docs.ovh.com/de/publiccloud/databases/backups/).
+
+Restoration of a backup is done by creating a new service and pushing the backup data to this new service. This full process is called forking and is fully automated. Once this process done you will have two indepedent service running, the one from which the backup come from, and a new one on which the backup data has been imported.
+
+## Instructions
+### Step 1: Select the database service you want to restore data from
+First you need to go on the overview page of the service you want to restore the backup from.
+
+![Select service](images/service-selection.png){.thumbnail}
+
+### Step 2: Go to the backup tab
+In the tab list click on `backups`{.action}.
+
+![service overview](images/service-overview.png){.thumbnail}
+
+### Step 3: Select the backup to restore from
+Select the backup from when you want to restore from. To help you choose you have the date at which the backups have been done in the colomn "Creation date".
+Click on the `...`{.action} button corresponding to the choosen backup. Then click on `Duplicate (Fork)`{.action} to go on the configuration page of the new service.
+
+> [!warning]
+> MongoDB service has the option to restore a backup in place. Meaning restoring the backup on the same service. This option will rollback ALL data to the state it was when the backup was done, this can induce data loss.
```

Comment:
Please rephrase:
The MongoDB service has the option to restore a backup in place, meaning restoring the backup on the same service. This option will rollback ALL data to the state it was in when the backup was done. This can induce data loss.